EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,81 @@
+## Editorial Style Suggestions
+This document provides editorial style suggestions for the FOCUS v1.0 specifications. It was created in response to a recent discussion about the need for a consistent and professional editorial style to enhance and harmonize FOCUS technical specifications. The guidelines outlined below aim to standardize the presentation of various elements such as **column names**, *column IDs*, [_glossary terms_](), **NORMATIVE KEYWORDS**, **`attribute names`**, *`attribute IDs`* and other key components of the FOCUS documentation.
+
+1. **Column Names**
+   - **Style**: **Bold**, (_with blank space for compound names_)
+   - **Reasoning**: Column names should be easily distinguishable for quick reference and clarity.
+   - **Example**: **Billing Account ID**, **Resource Name**
+
+2. **Column IDs**
+   - **Style**: *Italic*, (_without blank space for compound names, and compound words capatilized_)
```

Comment:
Should we specify how italics are written, given that they can be achieved via *single asterisk* or _underline_ or is it only what's rendered that matters?